README.md

```diff
@@ -320,8 +320,8 @@ function createErrorHandler (client, scope, config) {
   return (error, req, reply) => {
     if (config.dev) {
       console.error(error)
+      scope.vite.devServer.ssrFixStacktrace(error)
```

Comment:
in production it would end up crashing because it could not find a devServer under vite